
Fri, 01 Jan 2010 20:37:48 GMT

MAKE:
mv pidigits.python3-4.python3 pidigits.python3-4.py
/usr/local/src/Python-3.1.1/bin/python3.1 -OO -c "from py_compile import compile; compile('pidigits.python3-4.py')"
0.08s to complete and log all make actions

COMMAND LINE:
/usr/local/src/Python-3.1.1/bin/python3.1 pidigits.python3-4.pyo 2000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "pidigits.python3-4.py", line 24, in <module>
    main(int(sys.argv[1]))
  File "pidigits.python3-4.py", line 19, in main
    print(('%s\t:%d\n' * (len(out) / 2)) % tuple(out), end=' ')
TypeError: can't multiply sequence by non-int of type 'float'
